server/app/services/program/ProgramDefinition.java

```diff
@@ -655,7 +655,11 @@ private ImmutableList<BlockDefinition> getAvailablePredicateBlockDefinitions(
   }
 
   private static final ImmutableSet<QuestionType> NON_PREDICATE_QUESTION_TYPES =
-      ImmutableSet.of(QuestionType.ENUMERATOR, QuestionType.FILEUPLOAD, QuestionType.STATIC);
+      ImmutableSet.of(
+          QuestionType.ENUMERATOR,
+          QuestionType.FILEUPLOAD,
+          QuestionType.STATIC,
+          QuestionType.PHONE);
```

Comment:
@dkatzz  -  After a lot of back and forth, I decided not to include phone fore predicates. I did get it working but to make the countries show up in the dropdown UI, will require a lot more work and the product sn't looking for it now. I went to the Design doc and added a comment that "Predicates will not be handled for bare bones implementation". So I have added it here to filter this question type.

Reply:
Got it, thank you for the clarification!